osquery/tables/system/windows/processes.cpp

```diff
@@ -83,10 +101,15 @@ void genProcess(long pid, QueryData& results_data) {
       r["nice"] = INTEGER(lPlaceHolder);
       r["on_disk"] = osquery::pathExists(r["path"]).toString();
 
-      // TODO: some of these such as cwd, wired_size, phys_footprint
-      // should be retrievable either via Windows API or WMI
-      r["cwd"] = "";
-      r["root"] = "";
+      char fileName[MAX_PATH];
```

Comment:
We had been using `std::vector<char> filename(MAX_PATH);` then a `filename.data()` to access the buffer.
```cpp
std::vector<char> filename(MAX_PATH + 1);
filename.assign(MAX_PATH + 1, '\0');
```